package.json

```diff
@@ -687,6 +687,31 @@
               "url"
             ]
           }
+        },
+        "cursorless.experimental.hatStability": {
+          "description": "Determines how hard Cursorless should try to keep hats from moving around",
+          "type": "string",
+          "default": "low",
+          "enum": [
+            "low",
+            "lowRounded",
+            "lowThresholded",
+            "thresholded",
+            "high",
+            "highThresholded",
+            "higher",
+            "strict"
```

Comment:
try to orthogonalise into a threshold and then low, medium, high or something like that